market-service/src/main/java/rs/edu/raf/banka1/model/ListingModel.java

```diff
@@ -0,0 +1,48 @@
+package rs.edu.raf.banka1.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class ListingModel {
+    @Id
+    @JsonProperty("symbol")
+    private String ticker;
+
+    @JsonProperty("companyName")
+    private String name;
+
+    @JsonProperty("primaryExchange")
+    private String exchange;
+
+    @JsonIgnore
+    private LocalDateTime lastRefresh;
```

Comment:
takodje unix

Reply:
da li postoji razlog zasto je ovo jsonIgnore?

Reply:
jsonIgnore je, jer kada dobijem odgovor sa API-ja ne želim da mi on popuni to polje, već ga ručno popunjavam